frontend/main.go

```diff
@@ -35,6 +35,15 @@ func main() {
 	signalChannel := make(chan os.Signal, 1)
 	signal.Notify(signalChannel, syscall.SIGINT, syscall.SIGTERM)
 
+	// Verify the Async DB is available and accessible
+	logger.Info("Testing DB Access")
+	result, err := DBConnectionTest()
+	if err != nil {
+		logger.Error(fmt.Sprintf("Database test failed to fetch properties: %v", err))
+	} else {
+		logger.Info(fmt.Sprintf("Database check completed -- %v", result))
```

Comment:
I know this might be temporary, but we should structure the logging in general and not dump out a big string of escaped json. Maybe `Database check completed - %s", id` is enough since we don't really care about the rest of the properties?

Reply:
Agreed, we don't really need the properties, it more serves as proof of being able to capture something from the database without actually querying. Disappointed there isn't some kind of DB status call available in the package so this was to take its place. I'll get that updated. Thanks!

Reply:
@bennerv do we want to be using our custom cosmosDB library? I remember a conversation stating we'd try to only use the official library from Azure (since we likely won't need the changefeed feature), but I don't think it was documented.

Reply:
removed the marshaling call to just return the database id from the response directly instead.